shared/js/content-scripts/click-to-load.js

```diff
@@ -1,6 +1,8 @@
 (function clickToLoad () {
-    function sendMessage (messageType, options, callback) {
-        chrome.runtime.sendMessage({ messageType, options }, callback)
+    function sendMessage (messageType, options = {}) {
+        return new Promise((resolve, reject) => {
+            chrome.runtime.sendMessage({ messageType, options }, resolve)
```

Comment:
This doesn't handle .lastError like the mozilla shim does that we load elsewhere. Can we bundle that here instead?

Reply:
> This doesn't handle .lastError...

Yea, I nearly left a comment about this but decided against. My thinking was that we don't handle errors with the existing code either, I was trying to avoid changing the behaviour.

> the mozilla shim...

Could you point me at the existing shim you'd like me to include?

Reply:
I've tweaked `sendMessage` to handle lastError without the shim, does that look OK to you?

Reply:
https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/8d236939180663b07876d8f494cbbbf6c27f0b79/shared/js/background/wrapper.es6.js is an example that uses the shim. I don't think we have anything that exists in the content of the page currently and I think we can push that to a follow up as it probably requires build script wrangling that we should avoid here :)


Again can you file the follow up to clean this up into using the shim?

Reply:
Done.